packages/admin-ui-docs/docs/components/card.md

```diff
@@ -10,19 +10,20 @@ Cards are surfaces that display content and actions on a single topic.
 ## Usage
 
 ```jsx isStatic
-import { Card } from '@vtex/admin-ui'
+import { Card, Set, Heading, Paragraph } from '@vtex/admin-ui'
+
+;<Card>
+  <Set orientation="vertical" spacing={3} fluid>
+    <Heading>Title</Heading>
```

Comment:
Since this is more of a "usage" demo (the live example is below) that developers will probably copy I though maybe being explicit and concise was better. But I don't think we have a defined pattern for this 🤔  if you still think it should be a more realistic copy I can change it

Reply:
Guess we're good for now.